Makefile

```diff
@@ -106,6 +108,9 @@ ig-all: $(IG_TARGETS) ig
 ig: ig-$(GOHOSTOS)-$(GOHOSTARCH)
 	cp ig-$(GOHOSTOS)-$(GOHOSTARCH) ig
 
+debug-ig:
+	sudo IG_EXPERIMENTAL=true $(DLV) debug ./cmd/ig
```

Comment:
- Do we need to hardcode  IG_EXPERIMENTAL here?
- How can the user pass arguments to ig?
- Can we compile the binary without using sudo and then run dlv under sudo (by using go build + dlv exec)? to avoid issues with `go` not found for root and having to dowload all go modules for that user again.